packages/console/src/pages/AuthStatusChecker/index.tsx

```diff
@@ -0,0 +1,279 @@
+import { useLogto } from '@logto/react';
+import { yes } from '@silverhand/essentials';
+import { useCallback, useEffect, useState } from 'react';
+import { useTranslation } from 'react-i18next';
+
+import PageMeta from '@/components/PageMeta';
+import pageLayout from '@/scss/page-layout.module.scss';
+
+import { DEBUG_MODE_KEY } from './consts';
+import styles from './index.module.scss';
+import type { AdminTokenErrorMessage, AdminTokenStatusMessage, DebugLogEntry } from './types';
+import { AuthMessageType } from './types';
+import {
+  checkStorageAccess,
+  isCheckAdminTokenMessage,
+  isValidOrigin,
+  requestStorageAccess,
+} from './utils';
+
+/**
+ * Cross-domain authentication status checker page
+ * This page is designed to be embedded as an iframe to check login status
+ * across same-origin subdomains and localhost for development.
+ */
+function AuthStatusChecker() {
+  const { t } = useTranslation(undefined, { keyPrefix: 'admin_console' });
+  const [debugLogs, setDebugLogs] = useState<DebugLogEntry[]>([]);
+  const [isDebugMode, setIsDebugMode] = useState(false);
+  const [currentToken, setCurrentToken] = useState<string | undefined>(undefined);
```

Comment:
Just directly use `isAuthenticated` from SDK, right? Did you update this in the follow-up PRs?

Reply:
in debug mode interface, we show id tokens for debugging.